Core/clim-basic/geometry/region-predicates.lisp

```diff
@@ -66,12 +77,55 @@
   (every (lambda (r) (region-contains-position-p r x y))
          (standard-region-set-regions self)))
 
-(defmethod region-contains-position-p ((self standard-region-difference) x y)
-  (and (region-contains-position-p (standard-region-difference-a self) x y)
-       (not (region-contains-position-p (standard-region-difference-b self) x y))))
+(defmethod region-contains-position-p ((region standard-region-difference) x y)
+  (let ((region-a (standard-region-difference-a region))
+        (region-b (standard-region-difference-b region)))
+    (and (region-contains-position-p region-a x y)
+         (not (region-contains-position-p region-b x y)))))
+
+(defmethod region-contains-position-p ((self point) x y)
```

Comment:
Please use `region` instead of `self` for the newly added methods.

Reply:
do you want me to change also already existing methods? also, should I change transforms too to replace self with transformation?

Reply:
I would change it in any code I "touched".

Regarding the question whether the change makes sense: I think `self` is bad because

1. It is likely a result of the "classes own methods" line of thought
2. In the generic function definition the parameter is called `region` which accurately describes the role of the parameter in the generic function and protocol. I see no reason to deviate from that naming in methods.

Reply:
done